tools/c7n_azure/c7n_azure/provider.py

```diff
@@ -47,10 +47,20 @@ def initialize_policies(self, policy_collection, options):
         return policy_collection
 
     def get_session_factory(self, options):
+        cloud_endpoint = self.cloud_endpoints
+
+        # c7n-org will have a region set to either global or the specified region
+        region = options.get('region')
+        if region != '':
```

Comment:
This needs to be `if region:`. `options.get('region')` returns None if key doesn't exist.
Everything else looks good to me

Reply:
Updated!